### DISCUSSION TITLE
Working group catch up

### DISCUSSION THREAD

**Discussion by luckynvdu** [Sunbird-Ed/Community#65] [Announcements]
As part of the Sunbird ED community working group, we would invite all the members of the working group to discuss a couple of things -
1. Direction of Sunbird ED for the next 2 quarters
2. Alignment across different contributors
3. Define dedicated outcomes
4. Enabling Experience center
5. Evanglising Sunbird ED

Date - 4th Aug, 11.30 am - 12.30 pm
https://us06web.zoom.us/j/85233688438?pwd=VlpPUEJHUVBCNVpzamVhK1RxS0lhdz09

Meeting ID: 852 3368 8438
Passcode: 561350

@sharathkashyap @swayangjit @rajeevsathish @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @madhucr @alok-os

**Comment by aks30** [comment]
++ @vijiurs @kiranharidas187 @Shakthieshwari @Preethi-Sl

**Comment by baljeetsingh** [comment]
is this meeting happening ?

it shows meeting not started

CC : @coolbung @parthlawate

**Reply by coolbung** [reply]
@luckynvdu -- seems like the timing was changed later. Unfortunately github does not send any notifications for post edits, so we and probably many others missed the time change. Is there a recording / summary of the discussions that you can post here ?

**Comment by parthlawate** [comment]
Ah looks like the time changed.. We came in at 4 !  So missed it. 

We might want to use a better way to push event announcements that also allows for update notifications..  

A. Can someone post an MOM ? 
B. Is a follow up session planned ?

**Comment by luckynvdu** [comment]
@coolbung @baljeetsingh @parthlawate Apologies, We didn't realize that it didn't give notification on edit. It was changed due to the working group member's availability. Will take care in the future.

Please find the MOM and meet recording below - 

Hello everyone,

1. As a working group we need to define a 6 months roadmap based on the roadmaps from individual contributing organization
2. Contributors to come up with _suggestions and contributions_ on any specific workflow such as (but not limited to) admins in education or other domain, etc. to be enabled for the ED experience center at Ekstep
3. Alignment on what topics can be _included and contributed_ by contributor orgs for the Sunbird ED learning video series - https://github.com/Sunbird-Ed/Community/discussions/45

meet recording - https://drive.google.com/file/d/1sPn39jWNWhcW-4Vxq2BKBU_LPUYXxs5g/view?usp=sharing

We will be doing the follow-up session next week, and will keep the community posted here.

**Comment by luckynvdu** [comment]
Hello Everybody, we have planned for a Sunbird ED's working group follow-up catchup at 11.30 am - 12.30 pm, 11/08/2022

Use the link to join -

https://us06web.zoom.us/j/82646466602?pwd=S1ZUYi9ZbmJTV2pjT3lwRURoZzUyQT09

Meeting ID: 826 4646 6602
Passcode: 315080

@coolbung @parthlawate @baljeetsingh

**Comment by luckynvdu** [comment]
Hello Everybody, As part of today's meeting, the working group discussed upon the following action items for the next catchup-

1. Demo the Sunbird ED's revamped microsite for enabling adoption to the working group
2. Demo a Sunbird ED's instance in the professional development use case to the working group 
3. Learning series video demo to the working group 
4. Working group to discuss the 6-month roadmap from contributing organizations

Going forward, we will catch up once in two weeks. The next catchup will happen on 25th Aug at 11.30 am. 

https://us06web.zoom.us/j/82646466602?pwd=S1ZUYi9ZbmJTV2pjT3lwRURoZzUyQT09

Meeting ID: 826 4646 6602
Passcode: 315080

**Comment by luckynvdu** [comment]
Hello Everybody, please find the meet recording for the working group catchup on 25th Aug 2022 below -

https://drive.google.com/file/d/12zttRRd6Rno56YwCGYPemnjOKKWLY43z/view?usp=sharing

Details of the discussion are as follows -

1. @baljeetsingh and @parthlawate to come back on the Sunbird adoption efforts
2. Concretisation of roadmap taking inputs from contributors' roadmaps
3. Creation of roadmap items in the issue tracker as per the process by contributors

Joining link for next catchup -

https://us06web.zoom.us/j/81829249696?pwd=c0xqQkpSeGpmbGVtWno5UWQwZGIvUT09

Meeting ID: 818 2924 9696
Passcode: 504687

**Comment by parthlawate** [comment]
Thanks all . It was really exciting to see the Jenkins APK build process & refactored, configurable mobile app code today. Also @swayangit thanks for the detailed walk through .. 

We'd like to adopt this process for our adoption use cases. We have a few live scenarios that would help test & improve this process.. To enable that,  Is it possible to : 
1. Share a copy of the refactored app code removing any private keys etc ? 
2. Share Jenkins scripting so we can recreate the build process on our end ? 

We look forward to presenting the first draft of our gap analysis which we will contribute to the product backlog in the next meeting ! 

@nidhikachhawa24 @Xitija pls refer above video. Please plan to contribute in every way possible. Also apply this generalisation to web frontend @luckynvdu if there are similar efforts on for the web frontend, please loop us in so we can help

cc @baljeetsingh @coolbung @pramodkvarma @jagadishbabu

**Comment by luckynvdu** [comment]
Hello Everybody, please find the details of the discussion of the working group catchup on 8th Sep 2022 below -

@baljeetsingh and @parthlawate to come back on the Sunbird adoption efforts
SL and Tekdi to update the issue backlog in the new issue tracker by Monday 12th Sep
Clean up of the backlog to be completed by 15th Sep
Roadmap for Sunbird ED to be finalized by 22nd Sep (including the roadmap of all contributors)


Joining link for next catchup on 22nd Sep -

Join Zoom Meeting
https://us06web.zoom.us/j/82646466602?pwd=S1ZUYi9ZbmJTV2pjT3lwRURoZzUyQT09

Meeting ID: 826 4646 6602
Passcode: 315080

@Sunbird-Ed/sunbirded_workinggroup
